css/my_website.css

```diff
@@ -0,0 +1,52 @@
+body{
+    margin:0px;
+    background-color:rgb(231, 231, 231);
+    color:rgb(65, 65, 65);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size:20px;
+    line-height:normal;
+}
+.main-header{
+    background-color:#003F87;
+    color:#ffffff;
+    height: 75px;
+    padding: 0em;
+    width:100%;
+    overflow: hidden;
+}
+.text{
+    color: black;
+    padding:1em;
```

Comment:
Padding and left margin is not consistent. "Alejandro" is furthest left, "Hi, I'm" is further indented, then the remaining paragraphs are in-between the two.

Reply:
Now that I have them all on one class that is not a problem